modules/overview-of-openshift-gitops.adoc

```diff
@@ -0,0 +1,39 @@
+[id="overview-of-openshift-gitops"]
+= Overview of OpenShift GitOps
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: overview-of-openshift-gitops
+toc::[]
+
+:FeatureName: OpenShift GitOps
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Use Red Hat OpenShift GitOps to create repeatable processes for managing OpenShift Container Platform clusters and applications across multi-cluster Kubernetes environments. When you lose time in deployment and release cycles, Red Hat Openshift GitOps is a tool that handles and automates the fast pace and complex deployments. The Red Hat Openshift GitOps’ workflow pushes from application tp development environment to testing environment to staging to production.
+
+Red Hat Openshift GitOps is a declarative way to implement continuous deployment for cloud knative applications. Red Hat Openshift GitOps has a Git repository that contains a declarative description of the infrastructure you need in your specified environment. This Git repository also contains an automated process to make your environment match the described state. Basically,  Red Hat Openshift GitOps either deploys a new application or updates an existing one, so you only need to update the repository.  Red Hat Openshift GitOps handles and automates everything else!
+
+== How Red Hat Openshift GitOps works
+
+Red Hat OpenShift GitOps defines infrastructure and application definitions as “code.” Then, Red Hat OpenShift GitOps uses this code to manage multiple workspaces and clusters and to simplify the creation of infrastructure and application configurations. By following the code’s principles, you can store the configuration of clusters and applications in Git repositories then follow the Git workflow to apply these repositories to your chosen clusters. Red Hat OpenShift GitOps ensures consistency in applications when you deploy them to different clusters in different environments, like: development, stage, and production.
+
+Red Hat Openshift GitOps organizes the deployment process around the code repositories and makes them the central element. Red Hat Openshift GitOps always has at least two repositories:
+
+  . Application repository with the source code
+  . Environment configuration repository that defines the desired state of the application
```

Comment:
These sentences add more context here, suggest adding them here. Something like - 
These (please confirm which of the above repositories) repositories contain a declarative description of the infrastructure you need in your specified environment. It/They(?) also contain an automated process to make your environment match the described state.